signer/Cargo.toml

```diff
@@ -48,7 +48,7 @@ tokio = { workspace = true, features = ["signal", "macros", "rt-multi-thread"] }
 tokio-stream.workspace = true
 tracing.workspace = true
 tracing-attributes.workspace = true
-tracing-subscriber.workspace = true
+tracing-subscriber = { workspace = true, features = ["json", "fmt"]}
```

Comment:
These features, and more, are already pulled in.

Reply:
[ca98a84](https://github.com/stacks-network/sbtc/pull/920/commits/ca98a8401b5f2522f8b3120120ddabd6c16e63f8)